std_msgs/Header header
float64 x
float64 y
float64 width
float64 height
